README.md

```diff
@@ -1121,7 +1121,7 @@ import { getVehiclePositions } from 'gtfs';
 getVehiclePositions();
 ```
 
-### advancedQuery(table, query, fields, sortBy, join)
+### advancedQuery(table, advancedQueryOptions)
```

Comment:
I like this - do you think I should just update all node-gtfs query methods to use this format -then the `advancedQuery` would not be needed?

```
getStoptimes({
  query:  {
    trip_id: '37a',
  },
  fields: [],
  sortBy: [['stop_sequence', 'ASC']],
  join: [{
     type: 'INNER',
     table: 'trips',
     on: 'stop_times.trip_id=trips.trip_id'
   }],
  options: {}
});
```

Reply:
I think it is a good structure to make sure that there is no need for many arguments, however it would be a breaking change so perhaps it is too intrusive? 

I think the advancedQuery and runQuery and execQuery all are needed to give direct access to the database without opening additional connections depending on the scenario addressed.